pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
```

Comment:
I was about to disagree with you because F is not a vowel, but after some grammatical googling, I'll stand down.. 😉 👍